src/modules/75-cd/components/PipelineSteps/TerraformCloudRollbackStep/TerraformCloudRollback.tsx

```diff
@@ -0,0 +1,387 @@
+                    variableName="spec.discardPendingRuns"
+                    showRequiredField={false}
+                    showDefaultField={false}
+                    showAdvanced={true}
+                    onChange={value => setFieldValue('spec.discardPendingRuns', value)}
+                    style={{ alignSelf: 'center', marginTop: 11 }}
+                    className={css.addMarginTop}
+                    isReadonly={readonly}
+                  />
+                )}
+              </div>
+
+              <div className={cx(stepCss.formGroup, stepCss.md)}>
+                <FormInput.MultiTextInput
+                  name="spec.provisionerIdentifier"
+                  placeholder={getString('pipeline.terraformStep.provisionerIdentifier')}
+                  label={getString('pipelineSteps.provisionerIdentifier')}
+                  multiTextInputProps={{ expressions, allowableTypes }}
+                  disabled={readonly}
+                  className={css.addMarginTop}
+                />
+                {getMultiTypeFromValue(values.spec.provisionerIdentifier) === MultiTypeInputType.RUNTIME && (
+                  <ConfigureOptions
+                    value={values.spec.provisionerIdentifier as string}
+                    type="String"
+                    variableName="spec.provisionerIdentifier"
+                    showRequiredField={false}
+                    showDefaultField={false}
+                    allowedValuesType={ALLOWED_VALUES_TYPE.TEXT}
+                    showAdvanced={true}
+                    onChange={
+                      /* istanbul ignore next */ value => {
+                        setFieldValue('spec.provisionerIdentifier', value)
+                      }
+                    }
+                    isReadonly={readonly}
+                  />
+                )}
+              </div>
+            </>
+          )
+        }}
+      </Formik>
+    </>
+  )
+}
+
+const TerraformCloudRollbackInputStep: React.FC<TerraformCloudRollbackProps> = ({
+  template,
+  readonly,
+  path,
+  allowableTypes,
+  stepViewType
+}) => {
+  const { getString } = useStrings()
+  const { expressions } = useVariablesExpression()
+  const prefix = isEmpty(path) ? '' : `${path}.`
+  return (
+    <>
+      {isValueRuntimeInput(template?.timeout) && (
+        <div className={cx(stepCss.formGroup, stepCss.md)}>
+          <FormMultiTypeDurationField
+            multiTypeDurationProps={{
+              configureOptionsProps: {
+                isExecutionTimeFieldDisabled: isExecutionTimeFieldDisabled(stepViewType)
+              },
+              allowableTypes,
+              expressions,
+              disabled: readonly
+            }}
+            label={getString('pipelineSteps.timeoutLabel')}
+            name={`${prefix}timeout`}
+            disabled={readonly}
+          />
+        </div>
+      )}
+      {isValueRuntimeInput(template?.spec?.message) && (
+        <FormMultiTypeTextArea
```

Comment:
```suggestion
<div className={cx(stepCss.formGroup, stepCss.md)}>
        <FormMultiTypeTextArea
```
Is the wrapper classes not need here?

Reply:
No, this is as per mocks